dev-docs/bidders/viant.md

```diff
@@ -0,0 +1,36 @@
+---
+layout: bidder
+title: Viant
+description: Prebid Viant Bidder Adapter
+biddercode: viant
+media_types: banner, video, native
+gdpr_supported: true
+coppa_supported: true
+usp_supported: true
+gpp_supported: false
+schain_supported: false
+userId: uid2, unifiedId
+pbjs: true
+prebid_member: false
+dchain_supported: false
+safeframes_ok: false
+deals_supported: true
+floors_supported: true
+fpd_supported: false
+ortb_blocking_supported: false
+gvl_id: none
```

Comment:
Sorry, we can't let you claim to support GDPR without a GVL ID. Either add an ID or change gdpr_supported to false. thanks.

Reply:
changed gdpr_supported to false.